shared/src/libs/helpers/debounce/debounce.helper.ts

```diff
@@ -0,0 +1,11 @@
+import libraryDebounce from 'debounce';
+
+const DEBOUNCE_DELAY = 300;
+
+const debounce = (
+  function_: (payload: { search: string }) => void,
```

Comment:
This callback should not receive any arguments, because we should be able to use debounce helper for any callback

Reply:
Done